pkg/chartrenderer/default_test.go

```diff
@@ -87,4 +87,25 @@ var _ = Describe("ChartRenderer", func() {
 			Expect(actual).To(Equal(alpinePod))
 		})
 	})
+
+	Describe("#Manifest", func() {
+		It("should return manifest", func() {
+			chart, err := renderer.RenderEmbeddedFS(embeddedFS, alpineChartPath, "alpine", "default", map[string]string{})
+			Expect(err).ToNot(HaveOccurred())
+
+			manifests := chart.Manifest()
+			Expect(manifests).NotTo(BeNil())
+		})
+	})
+
+	Describe("#AsSecretData", func() {
+		It("should return renderet chart as secret data", func() {
+			chart, err := renderer.RenderEmbeddedFS(embeddedFS, alpineChartPath, "alpine", "default", map[string]string{})
+			Expect(err).ToNot(HaveOccurred())
+
+			data := chart.AsSecretData()
+			Expect(data).To(Not(BeNil()))
+			Expect(data["alpine_templates_alpine-pod.yaml"]).NotTo(BeNil())
```

Comment:
Should we also check if the contents of `data["alpine_templates_alpine-pod.yaml"]` match the expected yaml?